bin/fit_posparams

```diff
@@ -0,0 +1,383 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Analysis script comparing internally-tracked (theta, phi) to fvc-measured (x,y).
+The goal is to examine the general performance of all positioners during any
+move (not just a specialized calibration sequence).
+
+Data sources:
+
+    theta and phi:
+    --------------
+    The positioner moves database contains time-stamped data for values POS_T
+    and POS_P. These correspond to focalplane coordinates posintTP.
+    
+    x and y:
+    --------
+    The NERSC and KPNO servers contain numerous FITS files. These are processed
+    by desimeter. A result X_FP, Y_FP is generated from the centroids. This
+    corresponds to focalplane coordinates ptlXY.
+    
+    As of 2020-04-08, J. Guy identified 9472 such files from the commissioning
+    operations so far at the Mayall, and has run them through desimeter. See
+    Julien's email to desi-commiss@desi.lbl.gov of 2020-04-08, in which he
+    attaches several descriptive slides.
+
+    The fvc catalogs he produced are on NERSC at:
+        /global/cfs/cdirs/desi/engineering/fvc/desimeter/v0.2.0/
+
+    And the matched coordinates per positioner are on NERSC at:
+        /global/cfs/cdirs/desi/engineering/fvc/desimeter/v0.2.0/positioners/
+"""
+
+import os
+import sys
+import cProfile, pstats
+import pandas
+import math
+import numpy
+import time
+import datetime
+import multiprocessing
+import yaml
+
+# DESI-specific imports
+# path handling is to be improved as I migrate better into github/desimeter
+desimeter_path = os.path.abspath('../py/desimeter/')
+sys.path.append(desimeter_path)
```

Comment:
The intension for desimeter and other offline code is that users get `desimeter/py` into their `$PYTHONPATH` before running the code, and the code shouldn't manipulate `sys.path` internally.  In particular, this method of finding the library in `../py` relative to `bin/` works when running in a bare git checkout, but won't work in the future when desimeter is installed somewhere (in which case the relationship between the location of the scripts and the python libraries is system-dependent).

Reply:
Ok, will do that